src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -946,40 +952,42 @@ subroutine ocn_tracer_vmix_tend_implicit(meshPool, dt, vertDiffTopOfCell, layerT
       !$omp do schedule(runtime) private(N, A, B, C, rhs, tracersTemp, k)
       do iCell = 1, nCells
          ! Compute A(k), B(k), C(k) for tracers
+         Nsurf = minLevelCell(iCell)
          N = maxLevelCell(iCell)
 
          ! A is lower diagonal term
-         A(1)=0.0_RKIND
-         do k = 2, N
+         A(Nsurf)=0.0_RKIND
```

Comment:
Is there a slightly different treatment of zeroing-out unneeded entries in the tridiagonals between `tracer_vmix_tend_implicit` and `vel_vmix_tend_implicit`?

It looks like `A(1:Nsurf) = 0.0`, etc is done in `vel_vmix`, but only `A(Nsurf) = 0.0` in `tracer_vmix`. I think  there may only be a difference in unused values that are out of range, so I don't think this will impact results in any way.

Reply:
You're right. I didn't catch that because it is unused but we should be consistent. I'm happy to commit this change, but I might include it in phase 2 if including it here would hold up the merge since our tests are pretty much complete. @xylar or @mark-petersen do you have opinions about whether to commit this now or later?
```suggestion
         A(1:Nsurf)=0.0_RKIND
```

Reply:
I'm personally not worried about this, and think it can go into phase II --- I think the only difference would be that some array entries (that are never used) could be allocated with undefined values rather than being zeroed out.
 
This shouldn't happen in the `minLevelCell=1` case anyway though, as `A(1:Nsurf)` and `A(Nsurf)`, etc should do the same zeroing.